saskatoon/fixtures/init

```diff
@@ -37,7 +37,7 @@ ignore='saskatoon\.json'
 for fix in "${seq[@]}"
 do
     echo "Loading $fix"
-    ./loaddata $fix
+    . ./loaddata $fix
```

Comment:
what's the issue you were experiencing with this? Were the environment variables not passed properly to the loaddata script?

Reply:
`init` just  couldn't find the `loaddata` script. might be because environments work a little differently in nix, but AFAIK this method of calling works across diff systems.

Reply:
mmm I still don't understand why you would need to source the file to execute this script. does the `cd SCRIPT_DIR` directive work as expected?
Alternatively could we simply call the script using  `SCRIPT_DIR/loaddata` ?

Reply:
hmm, replacing with $SCRIPT_DIR/loaddata doesn't work either. 

definitely running in bash as the prob remains after running with:
- #!/bin/bash
- #!/usr/bin/env bash
- bash ./init

same error: file not found. echo prints out the right path. i dont exactly understand it either as my other scripts work fine calling the original way. 

gonna do some more tests